forgerock-am/amster/config/realms/root-openbanking/OAuth2Provider.json

```diff
@@ -27,7 +27,7 @@
     },
     "advancedOAuth2Config" : {
       "customLoginUrlTemplate" : "https://bank.&{ob.domain}:4201/login?goto=${goto}<#if acrValues??>&acr_values=${acrValues}</#if><#if realm??>&realm=${realm}</#if><#if module??>&module=${module}</#if><#if service??>&service=${service}</#if><#if locale??>&locale=${locale}</#if><#if request??>&request=${request}</#if>",
-      "supportedScopes" : [ "openid", "payments", "fundsconfirmations", "accounts" ],
+      "supportedScopes" : [ "openid", "payments", "fundsconfirmations", "accounts", "eventpolling" ],
```

Comment:
Only added to support it on local environments, this scope still set on [ob-kube-am-config](https://github.com/ForgeCloud/ob-kube-am-config) / [OAuth2Provider config](https://github.com/ForgeCloud/ob-kube-am-config/blob/274936701fd8a674ddcad3541ffeae80c0d49961/default/am/obri-650/realms/root-openbanking/OAuth2Provider.json) for deployments.